test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -675,15 +675,54 @@
                     (mt/formatted-rows [int int int])
                     first)))))))
 
+(mt/defdataset time-zone-edge-cases
```

Comment:
not too picky but how about calling them `datetimed-diff-edge-cases`?

Reply:
Good point. These are unlikely to be used for other tests. I'll call it `datetime-diff-time-zone-cases` to match the name of the test.